app/src/main/java/org/simple/clinic/summary/teleconsultation/TeleconsultationMessageBuilder.kt

```diff
@@ -33,6 +34,22 @@ class TeleconsultationMessageBuilder @Inject constructor(
         .appendln("https://app.simple.org/patient/${patientTeleconsultationInfo.patientUuid}")
         .appendln("")
 
+    message.appendln(resources.getString(R.string.patientsummary_contact_doctor_diagnosis))
+
+    val hyperTensionTitle = resources.getString(R.string.patientsummary_contact_doctor_diagnosis_hypertension)
+    val diagnosedWithHypertension = patientTeleconsultationInfo.medicalHistory.diagnosedWithHypertension
+    if (diagnosedWithHypertension.isAnswered) {
+      message.appendln("$hyperTensionTitle ${textForDiagnosisAnswer(diagnosedWithHypertension)}")
+    }
+
+    val diabetesTitle = resources.getString(R.string.patientsummary_contact_doctor_diagnosis_diabetes)
+    val diagnosedWithDiabetes = patientTeleconsultationInfo.medicalHistory.diagnosedWithHypertension
```

Comment:
A bug we found while reviewing: this should be using `diagnosedWithDiabetes` and not `diagnosedWithHypertension`.

Reply:
Oh yeah, my bad. Fixed it 👍